drupal-status-count.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Creates a toolbar item that lists the number of issues with different statuses.
+ *
+ * @type {{createElement: (function(): HTMLDivElement)}}
+ */
+const statusCount = {
+    createElement: function () {
+        const statusInput = document.getElementById("edit-assigned");
+        const statusText = statusInput.getAttribute("value").trim();
+        const statusFields = document.querySelectorAll(
+            "td.views-field.views-field-field-issue-status"
+        );
+        let statuses = [];
+        statuses = [
+            ...new Set(
+                [...statusFields].map((statusField) =>
+                    statusField.innerText.trim()
+                )
+            ),
+        ];
+
+        const statusesDiv = document.createElement("div");
+        statusesDiv.className = "statusCount";
+        statuses.forEach((name) => {
+            let statusIssue = { name: name, count: 0 };
+            statusFields.forEach((field) => {
+                if (field.textContent.includes(name)) {
+                    statusIssue.count++;
+                }
+            });
+            const statusDiv = document.createElement("div");
+            statusDiv.innerText = `${name}: ${statusIssue.count}`;
+            if (statusIssue.count) {
+                const clicker = document.createElement("a");
+                //clicker.setAttribute('href', '#')
+                clicker.setAttribute("user", name);
+                clicker.onclick = function (event) {
+                    const target = event.target;
+                    const targetIsFiltered = target.hasAttribute("filtered");
+                    document.querySelectorAll(".user-filter").forEach((link) => {
+                        link.innerText = "🔎";
+                        link.removeAttribute("filtered");
+                    });
+                    if (targetIsFiltered) {
+                        statusFields.forEach(
+                            (statusField) =>
+                                (statusField.closest("tr").style.display = "table-row")
+                        );
+                        return;
+                    }
+
+                    target.innerText = "✅";
+                    target.setAttribute("filtered", true);
+                    statusFields.forEach((statusField) => {
+                        const parentRow = statusField.closest("tr");
+                        if (statusField.innerText.includes(name)) {
+                            parentRow.style.display = "table-row";
+                        } else {
+                            parentRow.style.display = "none";
+                        }
+                    });
+                };
+                clicker.innerText = "🔎";
+                clicker.className = "user-filter";
+                statusDiv.appendChild(clicker);
+            }
+
+            statusDiv.className = `user-issue-cnt-${statusIssue.count}`;
```

Comment:
I don't think we need to add this class. The reason I added in the User case was because we the page was showing all the issues assigned to members of our scrum team like [this list](https://www.drupal.org/project/issues/search/automatic_updates?assigned=Wim%20Leers%2C%20tedbow%2C%20kunal.sachdev%2C%20omkar.podey%2C%20TravisCarden%2C%20yash.rode&project_issue_followers=&status%5B0%5D=1&status%5B1%5D=13&status%5B2%5D=8&status%5B3%5D=14&status%5B4%5D=4&status%5B5%5D=16&version%5B0%5D=all_8.x-2.%2A&issue_tags_op=%3D&issue_tags=sprint&text=&submitted=&&&&order=field_issue_status&sort=asc) then I wanted marked the users who have 0 issues as red. This way I would know to make sure to assign them an issue

With status you will never have 0 because you are only using the statuses on the page. and also having 0 issues for a particular status is not a problem